neuralmonkey/nn/mlp.py

```diff
@@ -6,26 +6,25 @@ class MultilayerPerceptron(object):
     """ General implementation of the multilayer perceptron. """
 
     # pylint: disable=too-many-arguments
-    def __init__(self, mlp_input, layer_configuration, dropout_plc,
+    def __init__(self, mlp_input, layer_configuration, dropout_keep_prob,
                  output_size, name: str = 'multilayer_perceptron',
-                 activation_fn=tf.nn.relu) -> None:
+                 activation_fn=tf.nn.relu, train_mode: tf.Tensor=None) -> None:
```

Comment:
train_mode by nikdy nemělo bejt None.. Za to ti možná už vynadalo mypy. Musí to být povinný parametr